config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  plugins: ['html'],
+  plugins: ['html', 'vue-libs'],
```

Comment:
These are recommended setting so I think it's perfectly fine to keep it there. Especially because you already set `'vue-libs/jsx-uses-vars': 2` in this config.

Btw. it's also what other plugins do in theirs recommended configurations e.g.: https://github.com/yannickcr/eslint-plugin-react/blob/master/index.js